### DISCUSSION TITLE
Filtering profile information

### DISCUSSION THREAD

**Discussion by EverlyScott** [pocketbase/pocketbase#633] [Q&A] [answered]
I am migrating over our existing "database" over to pocketbase and I'm trying to filter for specific users with their old user id generated before this switch. I have a `legacy_id` field in the users screen with the users' old ids set like this:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/31575858/192128769-0d345e74-8387-4dce-a37c-38d97f82a86e.png">
when I try to put in a filter for `legacy_id="2188ae130943750a4b70a5c9168ea759"` (bottom id in image), it returns no results. Why is this?

**Comment by EverlyScott** [comment]
the filter:
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/31575858/192128789-a2ea2a44-998a-4e8b-87c5-134f86a896fd.png">

**Comment by EverlyScott** [comment]
I also replicated this on the [live demo](https://pocketbase.io/demo/)

**Comment by ganigeorgiev** [comment, marked as answer]
The users search filter works only for the main user fields (id, email, verified, etc.). Profile fields aren't searchable from the UI at the moment.

This will be fixed with #376.
